### PR TITLE
Added possibility to select multiple files or jst one film/episode.

### PR DESCRIPTION
**Pull request by mihakodric**
Added possibility to select multiple files (maybe whole series) or just one film/episode. I also changed files selector to Tkinter, because it works well also on Windows.

### CODE REVIEW

**Comment by sameera-madushan** [comment]
I’m so sorry but some conflicts arose I love your pull requests do you think you could refactor it.

**Comment by mihakodric** [comment]
Thanks for the reply. This was my first pull request, so I hope everything is done correctly. Thank you for your understanding.

File: sbox.py [hunk 1]
```diff
@@ -9,16 +9,20 @@
...
+import numpy as np
 
 
 def tk_get_file_path():
     root = Tk()
     root.withdraw()
-    file_path = filedialog.askopenfilename()
+    
+    file_path = root.tk.splitlist(
+        filedialog.askopenfilenames(parent=root, title='Choose a file'))
```
Comment (sameera-madushan) [inline sbox.py:21]:
```suggestion
    if not file_path:
        print("Cancelled")
        sys.exit()

```

**Comment by sameera-madushan** [comment]
While i was testing your code i ran into a small issue. Imagine we have three movie files. for two of them SubDB have subtitles and one does not. Can you modify the code so that it will display "No Subtitle found" together with the movie movie name and dispay the available subs of the other two movies.

**Comment by mihakodric** [comment]
I followed your constructive comment, now the script works as you suggested.

**Comment by sameera-madushan** [comment]
Thanks for the contribution ♥

**Comment by sameera-madushan** [comment]
Hey @mihakodric 
It would be great if you can document (explain what they do) the code blocks that you've added.
(Please use the format already implemented in the file.)
Thank you.

**Comment by mihakodric** [comment]
Sure, I will do it as soon as I can.

**Pull request merged**